{"name": "test_login_invalid_user[test_info0]", "status": "broken", "statusDetails": {"message": "AttributeError: move_to requires a WebElement", "trace": "self = <test_cases.test_001_login.TestLogin object at 0x000001A62B8ED588>\ntest_info = {'expected': '请选择机构!', 'pwd': '123456', 'user': 'ceshiehdej'}\nget_browser = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"03babb195757e8f0daebebdcff7fa4c8\")>\n\n    @pytest.mark.parametrize('test_info', login_data_invalid)\n    @allure.story(\"用户名不存在\")\n    def test_login_invalid_user(self, test_info, get_browser):\n        print(\"------执行登录用户名不存在测试用例------\")\n        driver = get_browser\n        expected = test_info['expected']\n>       LoginPage(driver).login(test_info[\"user\"], test_info[\"pwd\"])\n\ntest_001_login.py:56: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\pages\\login_page.py:42: in login\n    self.move_click(self.btn)\n..\\common\\base_page.py:162: in move_click\n    webdriver.ActionChains(self.driver).move_to_element(ele).click(ele).perform()\nC:\\Python37\\lib\\site-packages\\selenium\\webdriver\\common\\action_chains.py:273: in move_to_element\n    self.w3c_actions.pointer_action.move_to(to_element)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.common.actions.pointer_actions.PointerActions object at 0x000001A62BF06AC8>\nelement = None, x = None, y = None\n\n    def move_to(self, element, x=None, y=None):\n        if not isinstance(element, WebElement):\n>           raise AttributeError(\"move_to requires a WebElement\")\nE           AttributeError: move_to requires a WebElement\n\nC:\\Python37\\lib\\site-packages\\selenium\\webdriver\\common\\actions\\pointer_actions.py:42: AttributeError"}, "attachments": [{"name": "log", "source": "ef8ac1fa-047b-4577-a70c-ccf8370ae7c4-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "test_info", "value": "{'user': 'ceshiehdej', 'pwd': '123456', 'expected': '请选择机构!'}"}], "start": 1600343976131, "stop": 1600344017083, "uuid": "9ac2c86b-1142-480b-90f3-d23747c573b4", "historyId": "075f4daf5ac51fdca2fb54fe12eff52f", "testCaseId": "cd0682f6a139d5392c369302963db253", "fullName": "test_cases.test_001_login.TestLogin#test_login_invalid_user", "labels": [{"name": "story", "value": "用户名不存在"}, {"name": "parentSuite", "value": "test_cases"}, {"name": "suite", "value": "test_001_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "LAPTOP-G986QGRS"}, {"name": "thread", "value": "16732-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_cases.test_001_login"}]}